CONTRIBUTING.md

```diff
@@ -108,7 +108,8 @@ to publish new versions of the image to public registries:
    - Commit these changes- `Bump version to x.y.z` is an acceptable commit
      message - and open a PR for review.
 1. Merge the version bump PR, and tag the repo with the new version.
-   - Ensure you are working from a current clone of the main branch
+   - Merge PR into main branch
+   - Ensure you are working from the up to date main branch
```

Comment:
There is no enforcement of rebase in this repository ?

Reply:
What does it mean?
Once you're merging PR you need to switch back to main locally and to pull it.

Reply:
I meant that github won't let merge if branch is not rebased. Like it in some project for example conjur oss. Seems but its not related so ok this instruction can stay.